introduction.md

```diff
@@ -0,0 +1 @@
+Git commit –am “git hub practice”
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
https://www.codeschool.com/courses/git-real